src/components/BaseSelect/BaseSelect.css.ts

```diff
@@ -54,16 +54,16 @@ export const listStyle = style([
     boxShadow: "overlay",
     borderColor: "neutralHighlight",
     width: "100%",
-    padding: "s1",
-    marginTop: "s1",
-    left: "s0",
-    maxHeight: "s52",
+    padding: 1,
+    marginTop: 1,
+    left: 0,
+    maxHeight: 52,
     overflowY: "auto",
   }),
   {
     borderRadius: calc.add(listItemBorderRadius, spaceBetweenListItemAndBorder),
     vars: {
-      [spaceBetweenListItemAndBorder]: vars.space[3],
+      [spaceBetweenListItemAndBorder]: vars.spacing[1],
```

Comment:
question: should be spacing[3]?

Reply:
It was leftover from the previous migration - it should be `s3`. It means that `vars.space[3]` had actually value of `4px` so it should be `1` in new scale.